packages/plugin-core/src/commands/CreateDailyJournal.ts

```diff
@@ -10,8 +10,12 @@ import {
 
 export class CreateDailyJournalCommand extends CreateNoteWithTraitCommand {
   constructor(ext: IDendronExtension) {
-    super(ext, "dendron.journal", new JournalNote());
-
+    super(
+      ext,
+      "dendron.journal",
+      new JournalNote(ext.workspaceService!.config)
```

Comment:
Should we explicitly check to make sure it's there instead of doing an non-null assertion?

Reply:
The command is gated behind the enablement clauses so we do know the workspace service will exist at that point, but doesn't hurt to be sure and give a proper error message than a stack trace.